src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/components/Announcements.tsx

```diff
@@ -62,6 +67,7 @@ const useStyle = StyleSheet.createThemeHook((theme) => ({
   scrollView: {
     paddingHorizontal: theme.spacings.medium,
     columnGap: theme.spacings.medium,
+    rowGap: theme.spacings.medium,
```

Comment:
Possible to just use `gap`, it should work both ways.

Reply:
I saw it when I created the pr, and kinda closed my eyes and hoping you woundn't notice 🙈 😆